lib/OpenCloud/Database/Service.php

```diff
@@ -104,4 +105,36 @@ public function datastoreList($params = array())
 
         return $this->resourceList('Datastore', $url);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setClient(ClientInterface $client)
+    {
+        // The Rackspace Cloud Databases service only supports the
+        // RC4 SSL cipher which is not supported by modern OpenSSL clients.
+        // Until the service can support additional, more modern and secure
+        // ciphers, this SDK has to ask curl to allow using the weaker
+        // cipher. For more information, see https://github.com/rackspace/php-opencloud/issues/560
+
+        $curlOptions = $client->getConfig()->get('curl.options');
+        $curlOptions['CURLOPT_SSL_CIPHER_LIST'] = static::getSslCipherList();
+        $client->getConfig()->set('curl.options', $curlOptions);
+
+        $logMessage = 'The SDK is using a custom cipher suite when connecting '
+                    . 'to the Rackspace Cloud Databases service. This suite contains '
+                    . 'a weak cipher (RC4) so please use at your own risk. See '
+                    . 'https://github.com/rackspace/php-opencloud/issues/560 for details.';
+        $client->getLogger()->critical($logMessage);
+
+        $this->client = $client;
+    }
+
+    public static function getSslCipherList()
```

Comment:
Is it okay if I add a link to Donald's comment (`https://github.com/rackspace/php-opencloud/issues/560#issuecomment-81790778`) in the docblock? I'm referencing the GitHub issue in other docblocks + the critical log message as well. The advantage of this is that the conversation in the GitHub issue provides more context than we could include inline in the code.